### ISSUE TITLE
Add option to hide `head & merge` section from status page

### ISSUE THREAD

**Issue by oguzbilgic** [tpope/vim-fugitive#1615]
I believe most people use Airline or something similar for status line, and they tend to show the current branch. For those cases the head and merge lines that are visible on the git status window are not really needed imho. I was wondering if adding an option to remove those lines would be a good idea, which would save 3 lines from the status buffer and make things a bit more minimalistic.

**Comment by tpope** [comment]
1. The current branch correlates with the `Head` line but not necessarily the `Merge` line.
2. Much to my chagrin, Airline uses the current work directory to determine the branch (iirc), so it won't be accurate if you've `cd`ed out of the repository.
3. There's also optional `Push` and `Bare` headers. I may add more headers in the future. I don't think we can remove the entire section all the time, and I think removing it some of the time would be bad.
4. There are actions you can invoke on the headers. Press `<CR>` on the `Head` header to open the topmost commit, for example.

tl;dr no
